client-misk/src/main/kotlin/app/cash/backfila/client/misk/embedded/Backfila.kt

```diff
@@ -9,29 +10,45 @@ import okio.ByteString
  * the Backfila dashboard UI.
  */
 interface Backfila {
-  fun <Type : Backfill<*, *>> createDryRun(
+  fun <Type : Backfill<*, *, *>> createDryRun(
     backfill: KClass<Type>,
     parameters: Map<String, ByteString>,
     rangeStart: String?,
     rangeEnd: String?
   ): BackfillRun<Type>
 
-  fun <Type : Backfill<*, *>> createWetRun(
+  fun <Type : Backfill<*, *, *>> createWetRun(
     backfillType: KClass<Type>,
     parameters: Map<String, ByteString>,
     rangeStart: String?,
     rangeEnd: String?
   ): BackfillRun<Type>
 }
 
-inline fun <reified Type : Backfill<*, *>> Backfila.createDryRun(
-  parameters: Map<String, ByteString> = mapOf(),
+inline fun <reified Type : Backfill<*, *, *>> Backfila.createDryRun(
+  parameters: Any? = null,
```

Comment:
I just started looking at the Management api, and realized I just want it to take in a Param class. Great timing